Gemfile

```
@@ -43,7 +43,7 @@ gem 'web', path: './engines/web'
 
 gem "activerecord-postgresql-adapter"
 gem "arel-helpers", "~> 2.12"
-gem "pg", "~> 1.2.3"
+gem "pg", "~> 1.2"
```

I reckon that we can drop that version constraint now entirely. That's just a relict from days before Dependabot when we were managing versions in the Gemfile.
```suggestion
gem "pg"
```